### PR TITLE
Handle auth error

### PR DESCRIPTION
**Pull request by mciparelli**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -24,8 +24,12 @@ const checkResponse = async response => {
     if (response.ok && isOK) return response.json();
     throw await response.json();
   }
-  if (response.ok && isOK) return response.text();
-  throw await response.text();
```
Comment (jkrehm) [inline src/index.js]: I believe removing this line was an accident. It should still throw if the response is not OK.

File: src/index.js [hunk 2]
```diff
@@ -24,8 +24,12 @@ const checkResponse = async response => {
...
+  if (response.ok && isOK) {
+    let responseText = await response.text();
```
Comment (jkrehm) [inline src/index.js]: Please switch to `const` because the values are not updated.

File: src/index.js [hunk 3]
```diff
@@ -24,8 +24,12 @@ const checkResponse = async response => {
...
+    let authFailed = responseText.includes('Authorization Error');
```
Comment (jkrehm) [inline src/index.js]: String.prototype.includes might not always be polyfilled, so IE 11 will fail on this line. Please switch to String.prototype.indexOf.

**Comment by mciparelli** [comment]
updated

**Pull request merged**